tensorflow_addons/image/filters.py

```diff
@@ -204,3 +204,57 @@ def median_filter2d(
         output = tf.cast(median, image.dtype)
         output = img_utils.from_4D_image(output, original_ndims)
         return output
+
+
+def gaussian_filter2d(
+    image: FloatTensorLike,
+    sigma: FloatTensorLike,
+    kernel_size: int,
```

Comment:
Please utilize the same parameter `filter_shape` as the other functions in the `filters.py` module. We need to be consistent so users can switch filters and not have a poor user experience.

Reply:
Done